geometry_msgs/Pose pose
bool planOnly
bool confirmation
